packages/web/app/package.json

```diff
@@ -9,12 +9,11 @@
     "typecheck": "tsc"
   },
   "dependencies": {
-    "@auth0/nextjs-auth0": "1.6.1",
     "@chakra-ui/react": "1.6.10",
-    "@emotion/css": "11.7.1",
-    "@emotion/react": "11.5.0",
-    "@emotion/server": "11.4.0",
-    "@emotion/styled": "11.6.0",
+    "@emotion/css": "11.10.0",
+    "@emotion/react": "11.10.0",
+    "@emotion/server": "11.10.0",
+    "@emotion/styled": "11.10.0",
```

Comment:
Had to bump these up because of conflicts with the emotion version used in the supertokens auth UI component.